examples/stress/index.html

```diff
@@ -23,7 +23,7 @@
 
 ///////////////////////////////////////////////////////////////////////////////
 
-class TestObject extends EngineObject 
+class TestObject extends EngineObject
```

Comment:
in the future, try to minimize adding white space like this, it helps greatly when reviewing changes. usually what i do is after i am ready to do a pull request i go through every line changed and make sure there is no unnessary changes

Reply:
My bad, will do 👍